stages/test/test_kickstart.py

```diff
@@ -241,6 +241,47 @@
     ({"ostreecontainer": {"transport": "dir", "url": "/run/install/repo/container", }, },
      "ostreecontainer --url=/run/install/repo/container --transport=dir",),
     ({"bootloader": {"append": "karg1 karg2=0"}}, "bootloader --append='karg1 karg2=0'"),
+
+    # %post
+    ({"%post": [{"commands": ["mkdir /scratch"]}]}, "%post\nmkdir /scratch\n%end"),
+    (
+        {
+            "%post": [
+                {"commands": ["mkdir /scratch"]},
+                {"commands": ["print('DONE!!!')"], "interpreter": "/usr/bin/python3"},
+            ]
+        },
+        "%post\nmkdir /scratch\n%end\n" +
+        "%post --interpreter \"/usr/bin/python3\"\nprint('DONE!!!')\n%end"
+    ),
+    (
+        {
+            "%post": [
+                {"commands": ["mkdir /scratch"]},
+                {
+                    "erroronfail": True,
+                    "nochroot": True,
+                    "interpreter": "/usr/bin/bash",
+                    "log": "/mnt/sysimage/var/log/ks-p2.log",
+                    "commands": [
+                        "echo 'Starting post2'",
+                        "if [ ! -e /mnt/sysimage/etc/resolf.conf ]; then",
+                        "  cp /etc/resolv.conf /mnt/sysimage/etc/resolve.conf",
```

Comment:
Are all the typos intentional? :rofl: 

`resolf.conf`, `resolve.conf`

Reply:
:sweat_smile: 

Just making sure everyone's paying attention >_>